SignalServiceKit/src/Storage/OWSStorage.m

```diff
@@ -457,7 +482,8 @@ + (YapDatabaseDeserializer)logOnFailureDeserializer
 
     return ^id(NSString __unused *collection, NSString __unused *key, NSData *data) {
         if (!data || data.length <= 0) {
-            return nil;
+            OWSProdLogAndFail(@"%@ can't deserialize null object: %@", self.logTag, collection);
+            return [OWSUnknownDBObject new];
```

Comment:
Just being defensive and trying to eliminate other possible causes of NPE.  But I don't think this is the issue after all.